firefox-ios/firefox-ios-tests/Tests/XCUITests/OnboardingTests.swift

```diff
@@ -321,4 +321,42 @@ class OnboardingTests: BaseTestCase {
         let topSites = app.collectionViews.cells[AccessibilityIdentifiers.FirefoxHomepage.TopSites.itemCell]
         mozWaitForElementToExist(topSites)
     }
+
+    func testFirstRunTourAccessibility() throws {
```

Comment:
@isabelrios Should we put such a test in TestRail and insert the TestRail link here?

Reply:
This test looks very similar to this one: https://github.com/mozilla-mobile/firefox-ios/blob/2a12d3b9ddf130e0742f2b3db2530386e6a02ee9/firefox-ios/firefox-ios-tests/Tests/XCUITests/OnboardingTests.swift#L29 
@thatswinnie are we missing something there? We can add it there so that we don't duplicate the complete test... Thanks!

Reply:
Oh never mind, just saw the call to `try app.performAccessibilityAudit()` we could add that in the exisiting test though...

Reply:
Discussed offline. We agreed to have this test as part of the exisiting UI `testFirstRunTour`. Then, the MTE team will create a specific test plan for a11y tests that can run as another workflow in parallel with the existing Smoketests and move this and other tests that will be implemented to it.

Reply:
FYi ^ @mdotb-moz